airbyte-integrations/connectors/source-facebook-marketing/unit_tests/test_async_job.py

```diff
@@ -334,15 +334,16 @@ def test_get_result_retried(self, mocker, job, api):
         api.call().json.return_value = {"data": [{"some_data": 123}, {"some_data": 77}]}
         ads_insights = AdsInsights(api=api)
         ads_insights._set_data({"items": [{"some_data": 123}, {"some_data": 77}]})
-        with mocker.patch(
+        mocker.patch(
```

Comment:
why do we need to remove context manager here?

Reply:
@artem1205, i am seeing the following warning when running unit tests with the previous version.

```
airbyte-integrations/connectors/source-facebook-marketing/unit_tests/test_async_job.py::TestInsightAsyncJob::test_get_result_retried
  xxx/airbyte-contributions/airbyte/airbyte-integrations/connectors/source-facebook-marketing/unit_tests/test_async_job.py:337: PytestMockWarning: Mocks returned by pytest-mock do not need to be used as context managers. The mocker fixture automatically undoes mocking at the end of a test. This warning can be ignored if it was triggered by mocking a context manager. https://pytest-mock.readthedocs.io/en/latest/remarks.html#usage-as-context-manager
    with mocker.patch(
```

Reply:
TIL, thanks for the explanation @h0rban !